src/helpers.js

```diff
@@ -0,0 +1,9 @@
+// A length of one signals that mapToProps does not depend on props from the parent component.
+// A length of zero is assumed to mean mapToProps is getting args via arguments or ...args and
+// therefore not reporting its length accurately..
+export function getDependsOnOwnProps(mapToProps) {
+  return mapToProps.dependsOnOwnProps !== null &&
+    mapToProps.dependsOnOwnProps !== undefined
```

Comment:
Not sure this part is needed but it seems it is provided by some libraries. So I kept it here. But we can also simplify this and just use
```
return mapToProps.length !== 1;

```